estate/views/estate_menus_views.xml

```diff
@@ -0,0 +1,12 @@
+<!-- views/estate_menus_views.xml -->
```

Comment:
While python header about encoding is not useful anymore, we tend to keep the one in xml (and this comment should not be here)